pyproject.toml

```diff
@@ -32,7 +32,8 @@ dependencies = [
   "click",
   "cryptography",
   "in-toto-attestation",
-  "sigstore",
+  "sigstore_protobuf_specs",
+  "sigstore~=4.0",
   "typing_extensions",
```

Comment:
i'm not as familiar with Python dependency operators, so please give this an extra look @mihaimaruseac
And would a dependency restriction like this warrant a changelog entry?

Reply:
Let's do `>=` for now and for sure let's also add to changelog. Thanks for flagging